cmd/migrate-from-testify/main_test.go

```diff
@@ -29,12 +20,6 @@ func TestRun(t *testing.T) {
 	defer env.Patch(t, "GO111MODULE", "off")()
 	defer env.Patch(t, "GOPATH", dir.Path())()
 
-	// Fetch dependencies in GOPATH mode
-	// Check list in testdata/full/some_test.go
-	goGet(t, "gopkg.in/check.v1")
-	goGet(t, "github.com/stretchr/testify/assert")
-	goGet(t, "github.com/stretchr/testify/require")
-
```

Comment:
@dnephin Why did you revert e4fe373a711e3d22e31783d0dbe76493816a77f1 ?

Reply:
I don't like the idea of having to `exec` the `go get` command from inside a test. There's nothing special about this test that should require running extra commands which are going to be relatively slow and will fail when there is no network.

I'd like to understand why only the Go1.19 job is failing in CI, that's really surprising to me.